File: JavaScript/Parameter Count/workspace/parameter_count_benchmark/cases/arg_variants.js
function mixedParameters(posOnly, regular, kwOnly, ...args) {
  return posOnly + regular + kwOnly + args.reduce((sum, value) => sum + value, 0);
}

function onlyVarargs(...items) {
  return items.reduce((sum, value) => sum + value, 0);
}

module.exports = { mixedParameters, onlyVarargs };

File: Python/Parameter Count/workspace/parameter_count_benchmark/cases/arg_variants.py
"""Benchmark case: positional-only, keyword-only, *args, **kwargs."""


def mixed_parameters(pos_only, /, regular, *args, kw_only, **kwargs):
    return pos_only + regular + sum(args) + kw_only + sum(kwargs.values())


def only_varargs(*items):
    return sum(items)


def only_kwargs(**options):
    return len(options)
